.github/workflows/docker-image.yml

```diff
@@ -117,7 +128,7 @@ jobs:
           context: .
           push: true
           file: ./Dockerfile
-          tags: "${{ steps.tags.outputs.value }}"
+          tags: "${{ github.event.inputs.tags || steps.tags.outputs.value }}"
```

Comment:
We use user provided tags if provided. Otherwise, we fallback to the old behaviour.

Reply:
Doesn't the build need the tag too ?

Reply:
Do you mean the other build steps in this workflow or the docker-build workflow?

If the former, they don't need custom tags because the tags they use are only used for local testing and then we push with different tags.

If the later, the docker-build job is only used for testing, so we're not using it for pushing. Also these two do not share artifacts. In fact, I think we should delete docker-build altogether and use this job for testing too (we'd need to make the push conditional here but that's not a problem).

Reply:
former